IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse6.mo

```diff
@@ -173,8 +178,104 @@ September 18, 2019 by Filip Jorissen:<br/>
 First implementation for the IDEAS crash course.
 </li>
 </ul>
-</html>"),
-    __Dymola_Commands(file=
-          "Resources/Scripts/Dymola/Examples/Tutorial/Example6.mos"
-        "Simulate and plot"));
-end Example6;
+</html>",
+info="<html>
+<p>
+This model extends <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse5\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse5</a> by adding a HVAC system.
+The system consists of a water-water heat pump, radiators, a storage tank, circulation
+pumps and a low temperature heat source for the heat pump. The model includes constant 
+control set points for the heat pump and pumps. A system is incorporated to measure 
+the electrical energy consumption of the heat pump.
+</p>
+<h4>Required models</h4>
+<ul>
+<li>
+<a href=\"modelica://IDEAS.Fluid.HeatPumps.ScrollWaterToWater\">
+IDEAS.Fluid.HeatPumps.ScrollWaterToWater</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Fluid.HeatExchangers.Radiators.RadiatorEN442_2\">
+IDEAS.Fluid.HeatExchangers.Radiators.RadiatorEN442_2</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Fluid.Actuators.Valves.TwoWayTRV\">
+IDEAS.Fluid.Actuators.Valves.TwoWayTRV</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Fluid.Movers.FlowControlled_dp\">
+IDEAS.Fluid.Movers.FlowControlled_dp</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Fluid.Sources.Boundary_pT\">
+IDEAS.Fluid.Sources.Boundary_pT</a>
+</li>
+<li>
+<a href=\"modelica://IDEAS.Fluid.Storage.Stratified\">
+IDEAS.Fluid.Storage.Stratified</a>
+</li>
+<li>
+<a href=\"modelica://Modelica.Blocks.Sources.IntegerConstant\">
+Modelica.Blocks.Sources.IntegerConstant</a>
+</li>
+<li>
+<a href=\"modelica://Modelica.Blocks.Continuous.Integrator\">
+Modelica.Blocks.Continuous.Integrator</a>
+</li>
+</ul>
+<h4>Connection instructions</h4>
+<p>
+The model components of <a href=\"modelica://IDEAS.Buildings\"> IDEAS.Buildings</a> have been
+integrated where possible. E.g. a zone can always have an occupancy schedule, or lighting models. However,
+for HVAC, many configurations exist in reality. Some zones have a radiator, which physically resides within the
+zone, while others may use floor heating, which is a property of the floor. Some zones may even have multiple
+radiators with different sizes, etc. Due to this multitude of options, HVAC components have not been integrated
+into the IDEAS.Buildings components (yet). Instead, they can be added manually by the user. This requires
+some knowledge by the user about how to interconnect the different components, and even about the relevant
+physical principles that are at play.
+</p>
```

Comment:
I would remove this section, as it has nothing to do with connection instructions.